tools/wptrunner/wptrunner/executors/executorchrome.py

```diff
@@ -21,6 +21,7 @@
     WebDriverTestDriverProtocolPart,
     WebDriverTestharnessExecutor,
     WebDriverTestharnessProtocolPart,
+    WebDriverWebExtensionsProtocolPart
```

Comment:
this import is probably not needed?

Reply:
nevermind, thought this was referring to the import in `executorwebdriver.py`. we will remove this